drivers/storage/portworx/portworx.go

```diff
@@ -236,6 +242,34 @@ func (p *portworx) PreInstall(cluster *corev1alpha1.StorageCluster) error {
 	return nil
 }
 
+func (p *portworx) validateEssentials() error {
+	if pxutil.EssentialsEnabled() {
+		resource := types.NamespacedName{
+			Name:      pxutil.EssentialsSecretName,
+			Namespace: api.NamespaceSystem,
+		}
+		secret := &v1.Secret{}
+		err := p.k8sClient.Get(context.TODO(), resource, secret)
+		if errors.IsNotFound(err) {
+			return fmt.Errorf("secret %s/%s should be present to deploy a "+
```

Comment:
should we add a message saying the user should go to PX-Central to generate one?

Reply:
There is no clear page on how/where they can generate the secret. I think once we have a doc page it would be easier to link it to that.
Looking at the above message hopefully they will look at the prereq section until then.